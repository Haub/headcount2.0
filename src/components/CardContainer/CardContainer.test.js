import React from 'react';
import { shallow, mount } from 'enzyme';
import { CardContainer } from './index';

describe('CARDCONTAINER', () => {
  let wrapper;
  const mockProps = {
    filteredDistricts: ['Academy 20']
  }
  beforeEach(() => {
    wrapper=shallow(<CardContainer {...mockProps} />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});