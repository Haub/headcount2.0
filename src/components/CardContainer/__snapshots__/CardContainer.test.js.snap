// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CARDCONTAINER should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    filteredDistricts={
        Array [
            "Academy 20",
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Unknown
          0="A"
          1="c"
          2="a"
          3="d"
          4="e"
          5="m"
          6="y"
          7=" "
          8="2"
          9="0"
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "0",
        "nodeType": "function",
        "props": Object {
          "0": "A",
          "1": "c",
          "2": "a",
          "3": "d",
          "4": "e",
          "5": "m",
          "6": "y",
          "7": " ",
          "8": "2",
          "9": "0",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Unknown
            0="A"
            1="c"
            2="a"
            3="d"
            4="e"
            5="m"
            6="y"
            7=" "
            8="2"
            9="0"
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "0",
          "nodeType": "function",
          "props": Object {
            "0": "A",
            "1": "c",
            "2": "a",
            "3": "d",
            "4": "e",
            "5": "m",
            "6": "y",
            "7": " ",
            "8": "2",
            "9": "0",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "disableLifecycleMethods": true,
    "hydrateIn": undefined,
  },
}
`;
